server/routes/user.routes.js

```diff
@@ -873,4 +874,41 @@ router.put(
  */
 router.get('/:id', hasValidObjectId, authenticate, isAdmin, UserController.getOneUser);
 
+/**
+ * @swagger
+ * /user/remittance-percentage:
+ *   put:
+ *     tags:
+ *       - User
+ *     description: Allows an admin to update the remittance percentage of a vendor
+ *     produces:
+ *       - application/json
+ *     requestBody:
+ *      content:
+ *        application/json:
+ *          schema:
+ *            type: object
+ *            properties:
+ *              vendorId:
+ *                  type: string
+ *              percentage:
+ *                  type: integer
+ *     responses:
+ *      '200':
+ *        description: Remittance percentage updated
+ *      '404':
+ *        description: User not found
+ *      '400':
+ *        description: Bad request
+ *      '500':
+ *       description: Internal server error
+ */
+router.put(
+  '/remittance-percentage',
```

Comment:
I believe we can just use remittance.
```suggestion
  '/remittance',
```